js/scenes/Speaker/SpeakerContainer.js

```diff
@@ -10,13 +12,37 @@ class SpeakerContainer extends Component {
       title: 'speaker',
     }
   }
+  
+  constructor(props) {
```

Comment:
If  `constructor` is just being to set up the internal state, you could remove it all together.

Reply:
Done